autopush/src/client.rs

```diff
@@ -863,7 +861,7 @@ where
                     Ok(endpoint) => {
                         data.srv
```

Comment:
silly question, but why not use a ref to the `srv` you just cloned above?

Reply:
good question actually. the Rc::clone above wasn't even needed apparently